App/Talent.App.WebApp/wwwroot/js/react/ReactScripts/Employer/ManageJob/ManageJob.jsx

```diff
@@ -58,27 +63,159 @@ export default class ManageJob extends React.Component {
        // your ajax call and other logic goes here
+       // http://localhost:51689/listing/listing/getSortedEmployerJobs?employerId=670dd32b7652cc0758907f2f&activePage=1&sortbyDate="desc"&showActive=true&showClosed=true&showDraft=true&showExpired=true&showUnexpired=true&limit=6
+       $.ajax({
+        url: link,
+        headers: {
+            'Authorization': 'Bearer ' + cookies,
+            'Content-type': 'application/json'
+        },
+        type: "GET",
+        contentType: "application/json",
+        dataType: "json",
+        data: {
+            activePage: this.state.activePage,
+            sortbyDate: this.state.sortBy.date,
+            showActive: this.state.filter.showActive,
+            showClosed: this.state.filter.showClosed,
+            showDraft: this.state.filter.showDraft,
+            showExpired: this.state.filter.showExpired,
+            showUnexpired: this.state.filter.showUnexpired,
+            limit: this.state.pageSize
+        },
+        success: function(response) {
+            let jobsData = null;
+            //console.log("response",response);
+            if (response.myJobs) {
+                jobsData = response.myJobs;
+                //console.log('jobsData', jobsData);
+            }
+            let pages = Math.ceil(response.totalCount / this.state.pageSize);
+            this.setState({loadJobs: jobsData, totalPages: pages}, () => {
+                if (callback) callback();
+            })
+        }.bind(this),
+        error: function (response) {
+            console.error('Error while retrieving jobs: ', response.status);            
+        }
+       });
     }
 
     loadNewData(data) {
-        var loader = this.state.loaderData;
+        let loader = this.state.loaderData;
         loader.isLoading = true;
-        data[loaderData] = loader;
-        this.setState(data, () => {
+        //update states that affect next loadData (filter, sort, page)
+        let newData = Object.assign({}, this.state, data);
+        this.setState(newData, () => {
             this.loadData(() => {
                 loader.isLoading = false;
                 this.setState({
-                    loadData: loader
+                    loaderData: loader
                 })
             })
         });
     }
 
+    handleSortFilter(event, evData) {
+        let name = evData.name;
+        let value = evData.value;
+
+        let stateVar = "";
+        if (name === "date") {
+            stateVar = "sortBy";
+        }
+        else{
+            name = value;
+            stateVar = "filter";
+            value = this.state.filter[name] ? false : true;
+        }
+
+        let newVarValue = {};
+        newVarValue[name] = value;
+        let stateVarValue = Object.assign({}, this.state[stateVar], newVarValue);
+        let data = {};
+        data[stateVar] = stateVarValue;
+        this.loadNewData(data);
+    }
+
+    handlePageChange(event, evData) {
+        console.log("evData", evData);
```

Comment:
Can you remove this console.log?

Reply:
I try to remove them after debugging, but sometimes I forget :-) Perhaps I should add some flag for development only?